### PR TITLE
DOCS-10220-create-and-implement-nav-partial

### PR DESCRIPTION
**Pull request by sarathecoubian**

### CODE REVIEW

File: modules/ROOT/pages/architecture-visualization.adoc [hunk 1]
```diff
@@ -25,7 +25,7 @@ You can hide nodes through what you select in the canvas:
 
 * To hide a single node, select it, and then click *Hide selected service*.
 * To hide multiple nodes, multiselect them, and then click *Hide selected services*.
-* To hide all of the services other than the selected nodes, you can click *Hide other services*. 
+* To hide all of the services other than the selected nodes, you can click *Hide other services*.
```
Comment (priyaayyar) [inline modules/ROOT/pages/architecture-visualization.adoc]:
```suggestion
* To hide all of the services other than the selected nodes, click *Hide other services*.
```

File: modules/ROOT/pages/customize-your-canvas.adoc [hunk 2]
```diff
@@ -4,14 +4,14 @@ include::_attributes.adoc[]
 endif::[]
 
 
-You can choose what appears in your Visualizer canvas by applying filters for environments, tags, and services.
+You can choose what appears in your Anypoint Visualizer canvas by applying filters for environments, tags, and services.
```
Comment (priyaayyar) [inline modules/ROOT/pages/customize-your-canvas.adoc:7]:
```suggestion
You can choose objects? in your Anypoint Visualizer canvas by applying filters for environments, tags, and services.
```

File: modules/ROOT/pages/customize-your-canvas.adoc [hunk 3]
```diff
@@ -4,14 +4,14 @@ include::_attributes.adoc[]
...
 This enables you to hide the nodes that you do not want to see in the application graph.
 Filters have a cascading design, so all nodes that don't belong to the checked environments or tags are automatically hidden from the list of services.
 
 You can toggle *Show dependents and dependencies* to see the services and dependencies of the nodes and services which have the selected tags.
 The *View* card and the canvas are updated to display the current selection.
 
-Visualizer editors can save defined views so that any Visualizer user can access them.
+Users who have the Visualizer Editor permission can save defined views so that any Anypoint Visualizer user can access them.
```
Comment (priyaayyar) [inline modules/ROOT/pages/customize-your-canvas.adoc:14]:
```suggestion
For Anypoint Visualizer users to be able to access defined views, Visualizer editors (with Visualizer Editor permission) must have first saved those defined views.
```

File: modules/ROOT/pages/index.adoc [hunk 4]
```diff
@@ -17,7 +17,7 @@ The connections between nodes are based on traffic between applications or APIs
...
+Understanding these dependencies has powerful implications, especially when you can overlay additional data about these apps onto the network diagram. The power of Anypoint Visualizer, and the visualizations tailored for Architecture, Troubleshooting, and Policies, is understanding this data in the context of your application network. From Anypoint Visualizer, you can inspect performance metrics or policy information, define views, arrange applications in layers, and customize the labels displayed for nodes and connections. See xref:customize-your-canvas.adoc[Customize the Canvas] for more information.
```
Comment (priyaayyar) [inline modules/ROOT/pages/index.adoc]:
```suggestion
Understanding these dependencies has powerful implications, especially when you can overlay additional data about these apps onto the network diagram. The power of Anypoint Visualizer and the visualizations tailored for Architecture, Troubleshooting, and Policies is in understanding this data in the context of your application network. From Anypoint Visualizer, you can inspect performance metrics or policy information, define views, arrange applications in layers, and customize the labels displayed for nodes and connections. For more information, see xref:customize-your-canvas.adoc[Customize the Canvas] .
```

File: modules/ROOT/pages/troubleshoot-visualizer.adoc [hunk 5]
```diff
@@ -1,6 +1,6 @@
 = Troubleshoot Visualizer
 
-If you encounter an error or issue while using Visualizer, you can try these solutions.
+If you encounter an error or issue while using Anypoint Visualizer, you can try these solutions.
```
Comment (priyaayyar) [inline modules/ROOT/pages/troubleshoot-visualizer.adoc]:
```suggestion
If you encounter an error or an issue while using Anypoint Visualizer, you can try these solutions.
```

File: modules/ROOT/pages/troubleshoot-visualizer.adoc [hunk 6]
```diff
@@ -9,13 +9,13 @@ If you encounter an error or issue while using Visualizer, you can try these sol
...
+|Anypoint Visualizer only supports specific Mule versions (runtimes).
+|If you are not seeing some of your applications in the Anypoint Visualizer canvas or in the services list, check your filters. If the filters are not the issue, you may need to update your applications to use a different Mule version if you want them to appear. View a list of xref:setup.adoc[supported Mule versions]. If you have upgraded to a Mule version 4 released after March 22, 2019, or Mule version 3.9 released after April 5, 2019, you must xref:monitoring::enable-apps-deployed-to-cloud.adoc[enable monitoring] before the applications and metrics will appear.
```
Comment (priyaayyar) [inline modules/ROOT/pages/troubleshoot-visualizer.adoc]:
```suggestion
|If you are not seeing some of your applications in the Anypoint Visualizer canvas or in the services list, check your filters. If the filters are not the issue, you may need to update your applications to use a different xref:setup.adoc[supported Mule version] for the applications to appear on the canvas. If you have upgraded to a version of Mule 4 that was released after March 22, 2019, or a version of Mule 3.9 that was released after April 5, 2019, you must xref:monitoring::enable-apps-deployed-to-cloud.adoc[enable monitoring] before the applications and metrics can appear.
```

File: modules/ROOT/pages/use-tags-in-visualizer.adoc [hunk 7]
```diff
@@ -11,7 +11,7 @@ If a tag is assigned to an application, the application will appear in every vie
 
 You can automate adding tags to nodes by using `anypoint.platform.visualizer.tags=<name>, <name>`, either in the application's properties file or at deployment time through the Runtime Manager UI or API. Note that this is an application property, not a server property.
 
-When you use this property, the nodes automatically show the assigned tags in the *View* card of Visualizer. You can add but cannot override tags in the UI; you can modify them only in properties. All letters in tag names are displayed in lowercase.
+When you use this property, the nodes automatically show the assigned tags in the *View* card of Anypoint Visualizer. You can add but cannot override tags in the UI; you can modify them only in properties. All letters in tag names are displayed in lowercase.
```
Comment (priyaayyar) [inline modules/ROOT/pages/use-tags-in-visualizer.adoc]:
```suggestion
When you use this property, the nodes automatically show the assigned tags in the *View* card of Anypoint Visualizer. You can add but cannot override tags in the UI. Additionally, you can modify the tags only in properties. All the letters in the tag names are displayed in lowercase.
```

File: modules/ROOT/pages/view.adoc [hunk 8]
```diff
@@ -3,7 +3,7 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-*Visualizer editors* can create auto-discoverable views that provide a single source of truth and that are accessible to their team members across environments.
+Users who have the Visualizer Editor permission can create auto-discoverable views that provide a single source of truth and that are accessible to their team members across environments.
```
Comment (priyaayyar) [inline modules/ROOT/pages/view.adoc]:
```suggestion
Users with the Visualizer Editor permission can create auto-discoverable views that provide a single source of truth and are accessible to their team members across environments.
```

File: modules/ROOT/pages/view.adoc [hunk 9]
```diff
@@ -13,20 +13,19 @@ Views display your saved filter selection, including:
 * New services deployed that match the selected environments and tags
 * If the *Show dependents and dependencies* option is selected and the a service is deployed and related to a selected service
 
-Views that you have access to appear in a list in the *View* card (click the drop-down arrow to see the list). This gives you easy access to all the views that are available to you, and saves you time because you don't need to readjust filters every time you access Visualizer to get the view that is relevant to you.
+Views that you have access to appear in a list in the *View* card (click the drop-down arrow to see the list). This gives you easy access to all the views that are available to you, and saves you time because you don't need to readjust filters every time you access Anypoint Visualizer to get the view that is relevant to you.
```
Comment (priyaayyar) [inline modules/ROOT/pages/view.adoc:16]:
```suggestion
For quick access and to save you from readjusting the filters every time you use Anypoint Visualizer, all the views to which you have access appear in a list in the *View* card (click the drop-down arrow to see the list). 
```

File: modules/ROOT/pages/view.adoc [hunk 10]
```diff
@@ -13,20 +13,19 @@ Views display your saved filter selection, including:
...
 
 == Defined Views
 
-As a Visualizer editor, you can create views that provide a consistent, high quality source of truth for the architecture of each of your organization's projects.
+If you have the Visualizer Editor permission, you can create views that provide a consistent, high quality source of truth for the architecture of each of your organization's projects.
```
Comment (priyaayyar) [inline modules/ROOT/pages/view.adoc]:
```suggestion
If you have the Visualizer Editor permission, you can create views that provide a consistent and high-quality source of truth for the architecture of each of your organization's projects.
```

File: modules/ROOT/pages/visualizer-app-network.adoc [hunk 11]
```diff
@@ -47,9 +47,9 @@ After selecting the organization and environment group, Anypoint Visualizer disp
 
 == Views
 
-Visualizer editors can define views based on filtered environments and tags. These defined views are auto-discoverable and available to all Anypoint Visualizer users who have permission for the included environments. These views can be opened in any of the use-case specific visualizations.
+Users who have the Visualizer Editor permission can define views based on filtered environments and tags. These defined views are auto-discoverable and available to all Anypoint Visualizer users who have permission for the included environments. These views can be opened in any of the use-case specific visualizations.
```
Comment (priyaayyar) [inline modules/ROOT/pages/visualizer-app-network.adoc]:
```suggestion
Users with the Visualizer Editor permission can define views based on filtered environments and tags. These defined views are auto-discoverable and are available to all Anypoint Visualizer users who have permission for the included environments. These views can be opened in any of the use-case-specific visualizations.
```

**Review by priyaayyar — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/customize-your-canvas.adoc (hunk 2 above)
Comment (sarathecoubian) [reply modules/ROOT/pages/customize-your-canvas.adoc:7]: Not choosing objects - just filtering using the parameters noted at the end.

File: modules/ROOT/pages/view.adoc (hunk 9 above)
Comment (sarathecoubian) [reply modules/ROOT/pages/view.adoc:16]: Written as it was to emphasize the UI object before anything else.

File: modules/ROOT/pages/customize-your-canvas.adoc (hunk 3 above)
Comment (sarathecoubian) [reply modules/ROOT/pages/customize-your-canvas.adoc:14]: Emphasizing permission requirement first since users kept missing this previously.

**Review by priyaayyar — APPROVED** [review]

**Pull request merged**